nuxt.config.js

```diff
@@ -21,7 +21,11 @@ module.exports = {
   ** Customize the progress bar color
   */
   loading: { color: '#3B8070' },
-  plugins: ['~/plugins/http-common.js', '~/plugins/prevent-scroll.js'],
+  plugins: [
+    '~/plugins/http-common.js',
+    '~/plugins/prevent-scroll.js',
+    '~/plugins/vue-markdown.js'
```

Comment:
As guide says we should also include vue-markdown as a vendor for better caching https://nuxtjs.org/guide/plugins